assets/js/components/HostDetails/HostSettingsPage.jsx

```diff
@@ -40,7 +47,7 @@ function HostSettingsPage() {
       selectedChecks={selectedChecks}
       catalog={catalog}
       catalogError={catalogError}
-      catalogLoading={catalogLoading}
+      loading={catalogLoading}
```

Comment:
I know that you do this to take advantage of the `...rest` thing, but it loses meaning in my opinion

Reply:
No strong preference, I thought it could make sense. I can revert that.

Reply:
removed the ...rest usage